R/board_connect.R

```diff
@@ -733,15 +733,25 @@ rsc_v1 <- function(...) {
 
 # Testing setup -----------------------------------------------------------
 
+# Connect testing: prefers using CONNECT_SERVER and CONNECT_API_KEY env vars,
+# but can fall back to Posit's demo PTD server if you've logged in with rsconnect
 board_connect_test <- function(...) {
-  if (connect_has_ptd()) {
-    board_connect_ptd(...)
+  if (nzchar(Sys.getenv("CONNECT_API_KEY"))) {
+    board_connect(auth = "envvar", ...)
+  } else if (connect_has_ptd()) {
```

Comment:
Thank you for working on this! Can you switch this back to the original order so we check for `connect_has_ptd()` first, for the local testing experience using rsconnect (vs. API keys)?

Reply:
Done in https://github.com/rstudio/pins-r/pull/884/commits/7ecbc48060b1fcf560f431858854d17731d321a2